packages/@lwc/engine-core/src/framework/stylesheet.ts

```diff
@@ -366,7 +366,8 @@ export function unrenderStylesheet(stylesheet: Stylesheet) {
         const abortController = cssContentToAbortControllers.get(cssContent);
         /* istanbul ignore if */
```

Comment:
```suggestion
        /* istanbul ignore if */
```

We can remove this since we have code coverage now. I verified that the `continue` is being hit. 🙂

Reply:
ah I forgot about this one.